app/build.gradle

```diff
@@ -1,4 +1,17 @@
+buildscript {
+    repositories {
+        maven {
+            url 'https://maven.fabric.io/public'
```

Comment:
Are we using Fabric ? I don't remember 👀

Reply:
Firebase Crashlytics uses Fabric internally.
https://firebase.google.com/docs/crashlytics/get-started